### ISSUE TITLE
White screen (Debian bookworm/testing Linux)

### ISSUE THREAD

**Issue by psow** [iotaledger/firefly#6201]
### Platform

Desktop

### Version

2.1.0

### Description

Blank white screen instead of the regular GUI.

### Expected behaviour

My wallet to be loaded and shown in the GUI

### Actual behaviour

During loading of the wallet screen becomes white (only the top menu is still there, but it doesn't work).

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. Run firefly
2. Choose profile/wallet
3. Enter the PIN
4. Screen becomes white during "syncing accounts..."

(note: initially firefly seemed to work normally, it broke when I tried to send a transaction - since then this issue always happens according to the steps above (reboot didn't help))

### Operating System

Linux

### Diagnostics

```shell
(this is before the issue happens - after it happens Diagnostics doesn't work)

Version 2.1.0
Language: en
Platform: linux
Platform Version: 6.1.0-5-amd64
Platform Architecture: x64
CPU Count: 24
Total Memory: 31939.0 MB
Free Memory: 26161.2 MB
```

### Developer Profile

No

### Profile Type

Hardware (Ledger)

### Staking

None

### Crash Status

White screen

### Error Messages

no

### Error Logs

```shell
Nothing shows up in the GUI, however this shows up in the console:

00:40:22.404 › Update for version 2.1.0 is not available (latest version: 2.1.0, downgrade is disallowed).
Sentry Logger [Log]: Found 0 minidumps
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at s.send (node:electron/js2c/browser_init:2:86358)
    at _.send (node:electron/js2c/browser_init:2:71969)
    at click (/tmp/.mount_fireflLA15Gu/resources/app.asar/public/build/main.js:2:585521)
    at MenuItem.click (node:electron/js2c/browser_init:2:30166)
    at a._executeCommand (node:electron/js2c/browser_init:2:35562)
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by psow** [comment]
I renamed the ~/.config/Firefly Shimmer directory and imported the wallet again from Ledger - this allowed me to see my wallet again.
However, when I tried to send SOON tokens it happened again (the same way I already described in my note in "steps to reproduce"). This time I noticed that this error showed up in the console at the moment when the screen turned white:

`thread '<unnamed>' panicked at 'arithmetic operation overflow', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/primitive-types-0.12.1/src/lib.rs:38:1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace`

I've run it again with "export RUST_BACKTRACE=1", but the same error printed only this time it suggested using RUST_BACKTRACE=full. When I did that, this is what I got:

`sadi@debian:~$ export RUST_BACKTRACE=full
sadi@debian:~$ Downloads/firefly-desktop-2.1.0.AppImage
xdg-mime: application argument missing
Try 'xdg-mime --help' for more information.
11:06:20.242 › Checking for update
11:06:20.293 › Generated new staging user ID: 75aa0719-57a8-5534-80bf-1080f38f0d91
11:06:20.453 › Update for version 2.1.0 is not available (latest version: 2.1.0, downgrade is disallowed).
thread '<unnamed>' panicked at 'arithmetic operation overflow', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/primitive-types-0.12.1/src/lib.rs:38:1
stack backtrace:
   0:     0x7f311c143b8a - <unknown>
   1:     0x7f311c16552e - <unknown>
   2:     0x7f311c13e895 - <unknown>
   3:     0x7f311c143955 - <unknown>
   4:     0x7f311c144eff - <unknown>
   5:     0x7f311c144c3b - <unknown>
   6:     0x7f311c145609 - <unknown>
   7:     0x7f311c145362 - <unknown>
   8:     0x7f311c14403c - <unknown>
   9:     0x7f311c1450b2 - <unknown>
  10:     0x7f311b375043 - <unknown>
  11:     0x7f311b84f8cc - <unknown>
  12:     0x7f311b8434a6 - <unknown>
  13:     0x7f311b88ec15 - <unknown>
  14:     0x7f311b791e01 - <unknown>
  15:     0x7f311b551b00 - <unknown>`

**NOTE:** I also tried sending SMR, $HIT and $EXY - all of them worked fine. For some reason, the problem is only when sending SOON tokens. It also doesn't seem to be dependent on expiration time or storage deposit gift settings for that transaction.

**Comment by psow** [comment]
I also tried with version 2.0.1 and I was able to send SOON tokens without any white screen issues. However, in 1 out of several tries this was printed in the console (but still, transaction went fine and GUI remained normal):
`11:18:45.517 › Update for version 2.0.1 is not available (latest version: 1.7.1, downgrade is disallowed).
thread '<unnamed>' panicked at 'arithmetic operation overflow', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/primitive-types-0.11.1/src/lib.rs:38:1
stack backtrace:
   0:     0x7f549d96f37d - <unknown>
   1:     0x7f549d99176c - <unknown>
   2:     0x7f549d969de1 - <unknown>
   3:     0x7f549d970b75 - <unknown>
   4:     0x7f549d970891 - <unknown>
   5:     0x7f549d971106 - <unknown>
   6:     0x7f549d970fb9 - <unknown>
   7:     0x7f549d96f874 - <unknown>
   8:     0x7f549d970d29 - <unknown>
   9:     0x7f549cb7df23 - <unknown>
  10:     0x7f549ce9721f - <unknown>
  11:     0x7f549cdf33fd - <unknown>
  12:     0x7f549ce66b90 - <unknown>
  13:     0x7f549cf4695e - <unknown>
  14:     0x7f549cfa0d14 - <unknown>
  15:     0x7f549cf010ce - <unknown>
  16:     0x7f549cf2f125 - <unknown>
  17:     0x7f549cd5644a - <unknown>
  18:     0x7f549d9746f3 - <unknown>
  19:     0x7f54a8b63fd4 - <unknown>
  20:     0x7f54a8be466c - <unknown>
  21:                0x0 - <unknown>`

**Comment by rheinschiene** [comment]
I have the same problem using Firefly 2.1.0 on Ubuntu 22.04. I tried to manage my vote power under the governance tab and after confirming the dialog to change the vote power, I got a request on my ledger to confirm the transaction. I could see the transaction in Firefly also for a very short moment telling me to review the transaction on my ledger and then the screen turned white.

This seems to destroy the Firefly profile, because I'm not able to load the profile anymore, without getting this white screen. After loading a backup of the profile, it works again until I try to change the vote power again.
